qiskit_ibm_runtime/options/resilience_options.py

```diff
@@ -48,10 +44,9 @@ class ResilienceOptions:
             Only applicable for ``resilience_level=2``.
             Default: ``None``, and (1, 3, 5) if resilience level is 2.
 
-        noise_amplifier (DEPRECATED): A noise amplification strategy. One of ``"TwoQubitAmplifier"``,
-            ``"GlobalFoldingAmplifier"``, ``"LocalFoldingAmplifier"``, ``"CxAmplifier"``.
-            Only applicable for ``resilience_level=2``.
-            Default: "TwoQubitAmplifier".
+        noise_amplifier (DEPRECATED): A noise amplification strategy. Currently only
+        ``"LocalFoldingAmplifier"`` is supported Only applicable for ``resilience_level=2``.
+            Default: "LocalFoldingAmplifier".
```

Comment:
We don't really need an option if there is only 1 choice. And it shouldn't be marked as `DEPRECATED` without a deprecation warning.

Reply:
Indeed the `DEPRECATED` should have been removed.
At the time, we left `noise_amplifier` as an option because it can be None, for resilience level lower than 2. Do you think we should remove it entirely, assuming the server knows to set it correctly? Or else keep as an option with just one choice that is hard coded?

Reply:
@merav-aharoni Well from UX perspective, we should remove it. However, you are correct that it looks like the server side primitive would blow up if it's not passed in 😞. We can remove it after it's fixed on the server side.